pom.xml

```diff
@@ -470,6 +470,7 @@
         <configuration>
           <consoleOutput>true</consoleOutput>
           <excludes>
+            <exclude>.github/PULL_REQUEST_TEMPLATE.md</exclude>
```

Comment:
Can't markdown files include XML comments with a license header?

Reply:
Yes, my preference would be to add the license to the file as well. However, there is a good reason to do not do this. This will show up in the editor when a user opens a PR. I think this hinders usability. WDYT. If you insist, I can open up a new PR, let me know.